conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureErrors.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.lib.internal;
+
+import com.palantir.logsafe.Safe;
+
+public final class ConjureErrors {
+    private ConjureErrors() {}
+
+    // This is used in generated code when generating errors which don't have any parameters.
+    public static final EmptyErrorParameters EMPTY_ERROR_PARAMETERS_INSTANCE = new EmptyErrorParameters();
+
+    public record EmptyErrorParameters() {}
+
+    public abstract static class BaseEndpointError<T> {
```

Comment:
This should live in the `dialogue-target` module within dialogue, ideally as a top-level class to keep the code a bit more concise